src/components/Sidebar/AddBranchNode.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { FC } from 'react';
```

Comment:
Add useState and ChangeEvent as an import here 
```suggestion
import React, { FC, useState, ChangeEvent } from 'react';
```